test/framefusion.test.ts

```diff
@@ -137,7 +137,7 @@ describe('FrameFusion', () => {
 
         // Act and Assert
         for (let i = 0; i < 10; i++) {
-            const frame = await extractor.getFrameAtTime(i / FPS + FRAME_SYNC_DELTA);
+            const frame = await extractor.getFrameAtTime(i / FPS);
```

Comment:
@stepancar question: is this equivalent to how we compute time in Luminary?

Reply:
@antoineMoPa , yes. The same time framefusion implementation shouldn't be aware of how it used in luminary.
I just fixed basic case when external code requests frame for time which points exactly to frame start in video.

Reply:
Makes sense!